sqlsimplekv/driver.go

```diff
@@ -0,0 +1,97 @@
+package sqlsimplekv
+
+import (
+	"bytes"
+	"database/sql"
+	"strings"
+	"text/template"
+
+	errgo "gopkg.in/errgo.v1"
+)
+
+type tmplID int
+
+const (
+	_ tmplID = iota - 1
+	tmplGetKeyValue
+	tmplGetKeyValueForUpdate
+	tmplInsertKeyValue
+	numTmpl
+)
+
+type queryer interface {
```

Comment:
Should it be ExecContext, QueryContext, QueryRowContext ?

Reply:
I meant to add this one, thanks.

Reply:
done